packages/btcindexer/db/migrations/0001_initial_schema.sql

```diff
@@ -75,3 +75,26 @@ CREATE TABLE IF NOT EXISTS nbtc_addresses (
   is_active INTEGER NOT NULL DEFAULT TRUE,
   UNIQUE(btc_address, btc_network)
 ) STRICT;
+
+CREATE TABLE IF NOT EXISTS nbtc_utxos (
+    sui_id TEXT NOT NULL,
```

Comment:
there is a conflict between this and table in the other PR

Reply:
This PR is in place of the other PR, the table is copied. We will not merge the other one probably